src/bin/touch.ts

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+import * as program from "commander";
+import * as log4js from "log4js";
+
+import * as utils from "../utils";
+
+const logger = log4js.getLogger("touch");
+
+interface ProgramParams {
+  file: string;
+}
+
+program
+  .usage("<file>")
+  .parse(process.argv);
+
+const params = program as any as ProgramParams;
+params.file = program.args[0];
+
+utils.writeFile(program.args[0], "", {flag: "wx"}).catch(error => {
```

Comment:
The `flag` parameter should not be hard-coded, but configurable (maybe with a default parameter).